parsely/src/test/java/com/parsely/parselyandroid/LocalStorageRepositoryTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.parsely.parselyandroid
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import java.io.File
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class LocalStorageRepositoryTest {
+
+    private lateinit var sut: LocalStorageRepository
+    private val context = ApplicationProvider.getApplicationContext<Context>()
+
+    @Before
+    fun setUp() {
+        sut = LocalStorageRepository(context)
+    }
+
+    @Test
+    fun `when expelling stored event, then assert that it has no effect`() {
+        // given
+        sut.persistQueue((1..100).map { mapOf("index" to it) })
+
+        // when
+        sut.expelStoredEvent()
+
+        // then
+        assertThat(sut.getStoredQueue()).hasSize(100)
+    }
```

Comment:
This test proves that `expelStoredEvent` has no effect on the stored list. I'm thinking we should remove this logic from the SDK. Internal communication: p1699018266041689-slack-C0533SEJ82H